sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
                 self.do_check[instance_key] = True
 
                 # check to see if the database exists before we try any connections to it
-                with self.open_managed_db_connections(instance, None, db_name=self.DEFAULT_DATABASE):
+                with self.open_managed_db_connections(instance, None, db_name=instance.get('database')):
```

Comment:
If we are hoping to enable a connection string with no `database` component, shouldn't we continue to have the config option disabled by default?  That way this `instance.get` call will return `None`.

Reply:
I still think the default should be `master` so disabling it should be a deliberate action.